src/hooks/useFarming.ts

```diff
@@ -176,7 +176,6 @@ type UseFarming = {
 export function useFarming({ lpContract, stakingContract }: UseFarming = {}): Farming {
   const [farmApr, setFarmApr] = useState('0')
   const [lpApr, setLpApr] = useState('0')
-  const [totalApr, setTotalApr] = useState('0')
```

Comment:
Why did you decide against using state for this number? returning the computed value cause this to be re-computed on every re-render of any component that imports this hook

Reply:
This was a solution provided to me by @bootsboogie

Reply:
@unclerom My thought was that the only time the total changed was when ever the farm or the lp apr changed. And since we needed to get this to work with and with out the staking contract I figured the logic would be complex to tell when we should actually set the total since we are setting the aprs in different if blocks. And I figured we did not need it to trigger a rerender if a rerender is already triggered by updating the individual aprs. I could go either way on it though. That was just my first thought to get this thing working without the staking contract.